src/main/resources/schema/query.graphqls

```diff
@@ -1,4 +1,17 @@
 type Query {
+  "Get a wish by ID"
   wish(id: ID): Wish
-  allWish: [Wish!]!
+
+  "Gets all Wishes"
+  allWishes: [Wish!]!
+
+  "Get all Wishes for Whiser"
+  myWishlist(userID: ID): [Wish]
```

Comment:
Always return an array
myWishlist(userID: ID): [Wish]!

Reply:
Yeah, the array should be always returned - []!

I'd also argue that wishes inside an array should not be null - [Wish!]!

http://spec.graphql.org/June2018/#sec-Combining-List-and-Non-Null